python/tank_vendor/shotgun_authentication/ui_authentication.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Ui based authentication.
+"""
+
+from .errors import AuthenticationCancelled
+
+import logging
+
+logger = logging.getLogger("shotgun_authentication").getChild(
```

Comment:
Is this a little OTT? Maybe just `logging.getLogger("shotgun_auth.ui")`. What value does this granularity add?